ambrs/aerosol.py

```diff
@@ -15,6 +15,22 @@
 # (this frozen type isn't made available by the scipy.stats package)
 RVFrozenDistribution = TypeVar('RVFrozenDistribution')
 
+class Delta(scipy.stats.rv_continuous):
```

Comment:
If you're interested, you can compare with the implementation in [this package](https://github.com/JohannesBuchner/fasterscipydists).